scripts/lamplib/tasks/run_tests.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
This file and `test_run_tests.py` are the only two pieces of new Python code. The rest of the Python files is mostly splitting up the original script.